pandas/core/algorithms.py

```diff
@@ -1448,6 +1448,27 @@ def func(arr, indexer, out, fill_value=np.nan):
     return func
 
 
+def take(arr, indexer, fill_value=None, allow_fill=None):
```

Comment:
reason to use `fill_value=None` instead of `fill_value=True` ?

Reply:
I assume you mean `allow_fill=True`? It should be a bool, not None.

What do we want the default to be? I thought `allow_fill=False`, to be compatible with ndarray makes sense.

Reply:
yes, sorry, both correct :-) I meant `allow_fill`, and `False`